packages/engine/Source/Core/sampleTerrain.js

```diff
@@ -132,6 +132,10 @@ function doSampling(terrainProvider, level, positions) {
   const tileRequestSet = {}; // A unique set
   for (i = 0; i < positions.length; ++i) {
     const xy = tilingScheme.positionToTileXY(positions[i], level);
+    if (!defined(xy)) {
```

Comment:
In order to make sure the code still builds, make sure to import `defined` at the top of this file.

Reply:
done thanks